packages/generator/src/vdm-types.ts

```diff
@@ -64,11 +64,11 @@ export interface VdmServiceEntities {
   /**
    * @internal
    */
-  functionImports: VdmFunctionImport[];
+  functionImports: VdmOperation[];
   /**
    * @internal
    */
-  actionImports?: VdmActionImport[];
+  actionImports?: VdmOperation[];
```

Comment:
[q] Should we create a follow up to merge the actions and functions also on a Vdm level? We end up with a action-imports.ts and function-imports.ts in the generated clients and customers need to know if it is a action or function they want to import. We could also create a operations.ts which contains actions and functions. I think this would need some discussion - but we could get rid of the two request builders and many types.

Reply:
Perhaps create a follow up ticket.

Reply:
I like this idea. Just creating one file with all of them should be good enough.